packages/server/src/api/routes/apikeys.ts

```diff
@@ -1,12 +1,6 @@
-import Router from "@koa/router"
 import * as controller from "../controllers/apikeys"
-import authorized from "../../middleware/authorized"
-import { permissions } from "@budibase/backend-core"
+import { builderGroup } from "./endpointGroups"
 
-const router: Router = new Router()
-
-router
-  .get("/api/keys", authorized(permissions.BUILDER), controller.fetch)
-  .put("/api/keys/:key", authorized(permissions.BUILDER), controller.update)
-
-export default router
+builderGroup
```

Comment:
Is `group` the right name? Would this not be clearer as `builderRoutes` or similar?

Reply:
Yeah you're right, its basically just like the sub-routers, i'll update that now!